cloudmarker/test/test_vmdatadiskencryptionevent.py

```diff
@@ -0,0 +1,153 @@
+"""Tests for VMDataDiskEncryptionEventTest plugin."""
+
+
+import copy
+import random
+import unittest
+
+from cloudmarker.events import vmdatadiskencryptionevent
+
+base_record = {
+    'ext':  {
+        'cloud_type':  'azure',
+        'record_type':  'virtual_machine',
+        'subscription_id':  'sub_id_0000',
+        'all_data_disks_encrypted':  True
```

Comment:
I don't see the tests depending on `cloud_type`, `record_type`, and `subscription_id`, so I think these fields can be removed from `base_record`. The unit tests should be set up with as minimal data as possible, so that the tests are very focused and reading the tests also easily shows us what it is being that is tested.

Reply:
Removed `cloud_type`, `record_type` and `subscription_id`